packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -205,7 +204,7 @@ const helperBehavior = {
     const validNewPhases = validTransitions[phase];
     assert(
       validNewPhases.includes(newPhase),
-      `Vault cannot transition from ${phase} to ${newPhase}`,
+      X`Vault cannot transition from ${phase} to ${newPhase}`,
```

Comment:
I'm guessing a phase is a string constant from an enum. If so...

```suggestion
      X`Vault cannot transition from ${q(phase)} to ${q(newPhase)}`,
```

Reply:
done